lib/linkable.dart

```diff
@@ -94,29 +106,46 @@ class Linkable extends StatelessWidget {
     return _textSpans;
   }
 
-  _text(String text) {
+  TextSpan _text(String text) {
     return TextSpan(text: text, style: TextStyle(color: textColor));
   }
 
-  _link(String text, String type) {
+  TextSpan _link(String text, String type) {
     return TextSpan(
-        text: text,
-        style: TextStyle(color: linkColor),
-        recognizer: TapGestureRecognizer()
-          ..onTap = () {
-            _launch(_getUrl(text, type));
-          });
+      text: text,
+      style: TextStyle(color: linkColor),
+      recognizer: TapGestureRecognizer()..onTap = () => _onTap(text, type),
+    );
   }
 
-  _launch(String url) async {
+  void _onTap(String text, String type) {
```

Comment:
I think this function has not been used. Also, can you update the README, adding custom functions?